site/content/docs/features/config-file.md

```diff
@@ -1 +1,632 @@
+
+_**AdditionalECSServiceSecurityGroups**_  
+A comma-delimited list of EC2 security groups to assign to the ECS service.
+
+### Parameters for the `ConsoleAppEcsFargateService` recipe only
+---
+_**ECSServiceSecurityGroups**_  
+A comma-delimited list of EC2 security groups to assign to the ECS service.
+
+### Parameters for the `ConsoleAppEcsFargateScheduleTask` recipe only
+---
+_**Schedule**_  
+The schedule or rate (frequency) that determines when Amazon CloudWatch Events runs the task. For details about the format of this value, see [Schedule Expressions for Rules](https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html) in the [Amazon CloudWatch Events User Guide](https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/).
+
+## JSON syntax for AWS Elastic Beanstalk deployments
+
+Use this JSON definition to build and deploy an ASP.NET Core application to [AWS Elastic Beanstalk](https://docs.aws.amazon.com/elastic-beanstalk) on Linux (`RecipeId` set to "AspNetAppElasticBeanstalkLinux"). Recommended if you don't want to deploy your application as a container image.
+
+```
+{
+    "AWSProfile": "string",
+    "AWSRegion": "string",
+    "StackName": "string",
+    "RecipeId": "AspNetAppElasticBeanstalkLinux",
+    "OptionSettingsConfig":
+    {
+        "BeanstalkApplication":
+        {
+            "CreateNew": boolean,
+            "ApplicationName": "string"
+            "ExistingApplicationName": "string"
+        },
+        "BeanstalkEnvironment":
+        {
+            "EnvironmentName": "string"
+        },
+        "InstanceType": "string",
+        "EnvironmentType": "string",
+        "LoadBalancerType": "string",
+        "ApplicationIAMRole":
+        {
+            "CreateNew": boolean,
+            "RoleArn": "string"
+        },
+        "ServiceIAMRole":
+        {
+            "CreateNew": boolean,
+            "RoleArn": "string"
+        },
+        "EC2KeyPair": "string",
+        "ElasticBeanstalkPlatformArn": "string",
+        "ElasticBeanstalkManagedPlatformUpdates":
+        {
+            "ManagedActionsEnabled": boolean,
+            "PreferredStartTime": "string",
+            "UpdateLevel": "string"
+        },
+        "XRayTracingSupportEnabled": boolean,
+        "ReverseProxy": "string",
+        "EnhancedHealthReporting": "string",
+        "HealthCheckURL": "string,
+        "ElasticBeanstalkRollingUpdates":
+        {
+            "RollingUpdatesEnabled": boolean,
+            "RollingUpdateType": "string",
+            "MaxBatchSize": integer,
+            "MinInstancesInService": integer,
+            "PauseTime": "string",
+            "Timeout": "string"
+        },
+        "CNamePrefix": "string,
+        "ElasticBeanstalkEnvironmentVariables": keyvalue,
+        "VpcId": "string",
+        "Subnets": array,
+        "SecurityGroups": array

```

Comment:
For `Subnets` and `SecurityGroups`, the recipe file has: "Type": "List".
Is JSON type "array" correct for this?

Reply:
Yes, the Json array will be converted to either List<string> or SortedSet<string>

Reply:
ACK